apps/eew/sceewlog/sceewlog.py

```diff
@@ -227,6 +229,23 @@ def initConfiguration(self):
         except:
             pass
 
+
+        try:
+            self.eewScript = self.configGetString("EEW.script")
+            seiscomp.logging.info('EEW.script = %s' % self.eewScript )
+        except Exception as e:
+            seiscomp.logging.warning('Not possible to read EEW.script. Setting this to None')
+            self.eewScript = None
+
+        if self.eewScript is not None :
+            logprefix = '/tmp/sceewlog-script.'
+            print("%s test=1 > %slog 2> %serr &" % ( self.eewScript, 
+                                                           logprefix,
+                                                           logprefix ))
+            os.system( "%s test=1 > %slog 2> %serr &" % ( self.eewScript, 
```

Comment:
Fred, to avoid blocking the main thread, please, consider using subprocess instead of os.system

Reply:
Of course it is not blocked at all ! There is a `&` !!! It releases instantly!

Reply:
Hey, I have added my tests in the PR description ...

Reply:
> Of course it is not blocked at all ! There is a `&` !!! It releases instantly! Try it! And I don't want to add dependencies to the main sceewlog.

👍Okay, I didn't check that detail